scm/driver/bitbucket/webhook.go

```diff
@@ -598,6 +610,137 @@ type (
 		Repository  prCommentHookRepo        `json:"repository"`
 		Actor       prCommentHookUser        `json:"actor"`
 	}
+
+	pipelineHook struct {
+		Repository struct {
+			Type     string `json:"type"`
```

Comment:
can we re-use existing `repository` structure and `convertRepository` function?

Reply:
In case of bitbucket, payload struct is different. I have used the existing struct webRepository which is used in other event

Reply:
For bitbucket payload Json structure is different, i am using the existing webRepository struct used in other events.